packages/panzer/adapters-stk/src/Panzer_STK_ModelEvaluatorFactory_impl.hpp

```diff
@@ -602,8 +602,8 @@ namespace panzer_stk {
     // see if field coordinates are required, if so reset the workset container
     // and set the coordinates to be associated with a field in the mesh
     useDynamicCoordinates_ = false;
-    for(std::size_t p=0;p<physicsBlocks.size();p++) {
-      if(physicsBlocks[p]->getCoordinateDOFs().size()>0) {
+    for(std::size_t pb=0;pb<physicsBlocks.size();pb++) {
```

Comment:
For the next cleanup pass: If `physicsBlocks` is a standard container, prefer a loop of the form `for (auto&& block : physicsBlocks)`.